www/klei00/semester_project_1/styles/main.css

```diff
@@ -0,0 +1,184 @@
+* {
+    box-sizing: border-box;
+    font-family: Arial;
```

Comment:
Picasso z nás nikdo asi nebude, ale stálo by za to vyzkoušet více možností, třeba google font, obrázky na pozadí, průhlednost, flat-design, větší písmeno, najít si nějakou barevnou paletu... inspirací je na internetu mnoho :)

https://htmlcolorcodes.com/color-chart/
https://www.awwwards.com/websites/flat-design/